lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -306,16 +307,17 @@ object TransactionFactory {
         nonce: Felt,
         maxFee: Felt = Felt.ZERO,
         signature: Signature = emptyList(),
+        version: Felt = INVOKE_VERSION,
```

Comment:
Parameters with default values don't really work well in Java. Let's verify how this is translated to java. 
If it just requires explicitly passing version parameter every time, it's fine by me.